md/two-main-types-of-deployment.md

```diff
@@ -26,20 +26,15 @@ It is fast because the Bonita Portal and the Bonita Engine run on the same JVM a
 
 Even if the `bonita.war` comes with an embedded Bonita Engine, you can choose **not** to use it.
 
-In this deployment you must configure the JVM containing the client Bonita Portal to use the remote connection to the engine. To do so see [configuring the connection to a remote engine](configure-client-of-bonita-bpm-engine.md#client_config).
-
-::: info
-Don't forget to set in this JVM the following properties:
- * `org.bonitasoft.platform.username` must be set with the username of the platform admin
- * `org.bonitasoft.platform.password` must be set with the password of the platform admin
-:::
-
 ### Accessible through HTTP
 
 ![deploy2](images/images-6_0/poss_deploy2.png)
```

Comment:
It should be good to add to the picture a reference to the two application servers and then use it in the chapter. If we use "Bonita portal", this term should appear in the picture

Reply:
I don't understand `portal` is already present in all pictures of this page

![image](https://user-images.githubusercontent.com/27200110/59117451-0a040200-894e-11e9-8a4d-3759782b09e4.png)